### PR TITLE
Business model

### LINKED ISSUE #145: Should make it clear what our business model is
I've had some enquiries from folks about what our long-term business model is for JOSS. My answer is 'none', that is:

- We don't plan to have article processing charges (ever)
- We have clear policy on content licensing (CC-BY): http://joss.theoj.org/about#content_license

I'm thinking we should have a short section on the about page outlining this, perhaps with an overview of our running costs?

What do people think?

/ cc @openjournals/joss-editors

### PR DESCRIPTION
**Pull request by arfon**
This pull request adds a short extra section to the JOSS about page giving an overview of our running costs.

/ cc @openjournals/joss-editors for feedback.

Fixes #145

### CODE REVIEW

File: app/views/home/about.html.erb [hunk 1]
```diff
@@ -341,6 +342,22 @@
           <p>Astronomer exploring the role of data science in academia at <a href="http://escience.washington.edu/">University of Washington's eScience Institute</a>. Core contributor to <a href="http://scikit-learn.org/">scikit-learn</a> and other science-focused Python packages.</p>
         </div>
 
+        <div class="main" id="business_model">
+          <h1>Business Model</h1>
+          <p>The Journal of Open Source Software is an open access journal <em>committed to running at minimal costs, with zero publication fees (<a href="https://en.wikipedia.org/wiki/Article_processing_charge">article processing charges</a>) or subscription fees</em>.
+            With volunteer effort from our editorial board and community reviewers and minimal infrastructure costs we believe JOSS can remain a free community service.</p>
+
```
Comment (tracykteal) [inline app/views/home/about.html.erb]: The costs are minimal, but there are some expenses. Should we add that donations also keep the cost free for authors?

e.g. 'With volunteer effort from our editorial board, community reviewers, donations and minimal infrastructure costs ... '

Comment (arfon) [inline app/views/home/about.html.erb]: Yep, that's a good suggestion. I've updated with your suggested language.

File: app/views/home/about.html.erb [hunk 2]
```diff
@@ -341,6 +342,22 @@
...
+          <p>The Journal of Open Source Software is an open access journal <em>committed to running at minimal costs, with zero publication fees (<a href="https://en.wikipedia.org/wiki/Article_processing_charge">article processing charges</a>) or subscription fees</em>.
+            With volunteer effort from our editorial board, community reviewers, donations and minimal infrastructure costs we believe JOSS can remain a free community service.</p>
+
+          <p>In the spirit of transparency, below is an outline of our current running costs:</p>
+
+          <ul>
+            <li>Annual Crossref <a href="http://www.crossref.org/02publishers/20pub_fees.html">membership</a>: $275 / year</li>
+            <li>JOSS paper DOIs: $1 / accepted paper</li>
+            <li>JOSS website hosting (Heroku): $19 / month</li>
+          </ul>
```
Comment (arfon) [inline app/views/home/about.html.erb]: Everything's a donation from me right now...

Comment (katyhuff) [inline app/views/home/about.html.erb]: maybe we need a donation button...

Comment (arfon) [inline app/views/home/about.html.erb]: Yeah, wouldn't be a bad idea. Ever done something like that before?

Comment (katyhuff) [inline app/views/home/about.html.erb]: Yeah... let me look back into whether what we did for SCF could be translated to this context.

Comment (arfon) [inline app/views/home/about.html.erb]: :zap:

**Pull request merged**